cypher/cypher-docs/src/docs/dev/ql/administration/security/security-and-indexes-intro.asciidoc

```diff
@@ -73,15 +73,15 @@ This difference in results is due to the `OR` relationship between the two prope
 
 ==== Blacklisting properties
 
-Now consider blacklisting properties, like the `surname` property:
+Now consider denylisting/denying access on properties, like the `surname` property:
```

Comment:
Here I meant for you to select one of the options, either _denylisting_ **or** _denying access on_.